docs/stark-build/NG_CLI_BUILD_CUSTOMIZATIONS.md

```diff
@@ -245,6 +245,54 @@ If you desire to customize it, you can define it in the **baseHref** option of y
 }
 ```
 
+#### If you need to override `starAppMetadata`  or `starkAppConfig` depending on your target you can extends `indexTransform`
+
+```text
+{
+    ...
+    "projects": {
+        "your-app": {
+            ...
+            "architect": {
+                "yourTargetName": {
+                    ...
+                    "options": {
+                        ...
+                        "indexTransform": "path/to/your/new/config.js"  // default value: "./node_modules/@nationalbankbelgium/stark-build/config/index-html.transform.js"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+in your new index-transform
+
+```javascript
+"use strict"
+const starkHtmlTransformIndex = require("../node_modules/@nationalbankbelgium/stark-build/config/index-html.transform");
+
+const overridedData = {
+	starkAppConfig : {
+		// your app config
+	},
+	starkAppMetadata : {
+		// your app metadata 
+	}
+}
```

Comment:
I would recommend to extend the Stark common-data and override the desired properties:

```suggestion
const overriddenData = {
    ...
	starkAppConfig : {
		// your app config
	},
	starkAppMetadata : {
		// your app metadata 
	}
}
```